lib/registration.pm

```diff
@@ -931,6 +931,9 @@ sub scc_deregistration {
         quit_packagekit;
         wait_for_purge_kernels;
         assert_script_run('SUSEConnect --version');
+        # Remove registercloudguest when use suseconnect in a non public cloud environment.
+        my $cloudguest_bin = "/usr/sbin/registercloudguest";
+        assert_script_run "test -f $cloudguest_bin && rm -f $cloudguest_bin";
```

Comment:
Does this bin comes from the registration of public Cloud? If confirmed, then could we check whether the pcm is in SCC_ADDONS to remove it or not?

Reply:
> Does this bin comes from the registration of public Cloud? If confirmed, then could we check whether the pcm is in SCC_ADDONS to remove it or not?

[Yes](https://openqa.suse.de/tests/17359651#step/patch_sle/125), so need we remove `pcm` from SCC_ADDONS? currently it set `SCC_ADDONS=base,serverapp,python3,live,contm,lgm,pcm,wsm,ltss`.

Reply:
@lemon-suse now all VRs passed and please take a look again. Thanks